### ISSUE TITLE
Are you supposed to get same set of final ensemble when fitting a automl twice?

### ISSUE THREAD

**Issue by kaiweiang** [automl/auto-sklearn#517]
Hi again,

For these two runs, all the parameters are same including `seed=1`.

**Comment by mfeurer** [comment]
There is currently a bug in the randomization for the tie breaking. Ideally, the ensembles should be equal, but it turns out they aren't. Could you please try seeding numpy globally before running Auto-sklearn.

**Comment by mfeurer** [comment]
Yes, kind of.

However, with 66d9f09 the solution from #514 should make it deterministic. Please let me know if that's not the case. If it is this makes it deterministic, we should really add an example and make a unit test out of this.

**Comment by adithyabsk** [comment]
@mfeurer Just got around to testing it again detailed the results in the #514 thread... still no dice, unfortunately
